src/main/java/com/tabnine/balloon/GotItTooltipAction.java

```diff
@@ -0,0 +1,5 @@
+package com.tabnine.balloon;
+
+public interface GotItTooltipAction {
+  void onGotItClicked();
+}
```

Comment:
rename to `SuggestionHintTooltipAction`.
the function as well - `onSuggestionHintTooltipClicked`